packages/engine/Source/Scene/Tonemapper.js

```diff
@@ -1,52 +1,64 @@
 /**
  * A tonemapping algorithm when rendering with high dynamic range.
  *
- * @enum {number}
- * @private
+ * @enum {string}
  */
 const Tonemapper = {
   /**
    * Use the Reinhard tonemapping operator.
    *
-   * @type {number}
+   * @type {string}
    * @constant
    */
-  REINHARD: 0,
+  REINHARD: "REINHARD",
```

Comment:
Now that this property is public and not private I changed these to strings to make it easier to understand if someone happens to log or view the value of `viewer.scene.postProcessStages.tonemapper`. I believe it shouldn't have any functional impact